### PR TITLE
adds search bar for JSON and hardcoded nutrition

### PR DESCRIPTION
**Pull request by keithlwill**
I've added a text input and a button that will run a function to retrieve the JSON for text input. I've also added 3 hard coded buttons that will retrieve the JSON for the nutrition facts for predetermined foods. The JSON objects are printed to the console.

https://screenshot.googleplex.com/UhxjKuc4Dw2.png

### CODE REVIEW

**Review by CalebBaines1 — APPROVED** [review]

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -5,13 +5,24 @@
     <title>Capstone Project</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="nutrition.js"></script>
```
Comment (CalebBaines1) [inline src/main/webapp/index.html:9]: It is not best practice to use multiple separate JavaScript files as it increases the loading time of a page. Wherever possible you usually only want to have one. See this for further reading https://developer.yahoo.com/performance/rules.html?guccounter=1

File: src/main/webapp/nutrition.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+//Use user input to search for possible foods
```
Comment (CalebBaines1) [inline src/main/webapp/nutrition.js]:
```suggestion
/**
 * Use user input to search for possible foods.
 */
```
This is one of the standard ways to comment functions in JavaScript and this applies to the other functions you wrote as well. I omitted writing the arguments and return types. This is a good reference for how people write JavaScript at google https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

File: src/main/webapp/nutrition.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+function foodSearch() {
+    let userfood = document.getElementById('foodInput').value;
+    let url = 'https://trackapi.nutritionix.com/v2/search/instant?query=' + userfood;
+    let myHeaders = new Headers({ 
+        'x-app-id': '85f8ef5f',
+        'x-app-key': '7dac3e04e347a6dc70bfd89e404501d3'
```
Comment (CalebBaines1) [inline src/main/webapp/nutrition.js]: You should try to hide your api keys for various reasons. Here is one way to do that
https://gist.github.com/derzorngottes/3b57edc1f996dddcab25

File: src/main/webapp/index.html (hunk 1 above)
Comment (ruixia) [reply src/main/webapp/index.html:9]: +1, but to be precise this doesn't mean project developers implement Javascript code in one single file. 

The common practice is to develop the code in separate files, and minify all files into one file at the deployment phase :) Of course this is an optimization for large size production projects and it's an overkill for our capstone project.

File: target/portfolio-1/index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (ruixia) [inline target/portfolio-1/index.html:1]: This is a duplicate of webapp/index.html. Let's remove it?

File: target/portfolio-1/nutrition.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
+var apiID = config.NAppID;
```
Comment (ruixia) [inline target/portfolio-1/nutrition.js:1]: This is a duplicate of webapp/nutrition.js. Let's remove it?

File: target/portfolio-1/index.html (hunk 4 above)
Comment (ruixia) [inline target/portfolio-1/index.html:1]: target/portfolio-1 has duplicated files of /webapp/. Let's remove them?

**Pull request merged**